.azure-pipelines/templates/test-single-linux.yml

```diff
@@ -45,6 +45,7 @@ jobs:
       repo: ${{ parameters.repo }}
       pip_cache_config: ${{ parameters.pip_cache_config }}
       run_py2_tests: ${{ parameters.run_py2_tests }}
+      dd_url: ${{ parameters.dd_url }}
```

Comment:
This parameter must be defined at the top of the file under `parameters` with a default value.
You'll probably need to define and pass it in this file too https://github.com/DataDog/integrations-core/blob/master/.azure-pipelines/templates/test-all.yml since that's what you're using on  the marketplace master branch

Same for windows steps